libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationOnCameraTrackingChangedListener.java

```diff
@@ -0,0 +1,29 @@
+package com.mapbox.services.android.navigation.ui.v5;
+
+import android.support.design.widget.BottomSheetBehavior;
+
+import com.mapbox.mapboxsdk.plugins.locationlayer.OnCameraTrackingChangedListener;
+
+public class NavigationOnCameraTrackingChangedListener implements OnCameraTrackingChangedListener {
```

Comment:
Let's add javadoc for this new class.

Reply:
👀 the constructor is not public so wasn't sure - added regardless

Reply:
@danesfeder 
Ohh didn't 👀 In that case, I guess the `public` modifier is not needed, right?